src/query/config.hpp

```diff
@@ -26,5 +26,8 @@ struct InterpreterConfig {
   std::string default_pulsar_service_url;
   uint32_t stream_transaction_conflict_retries;
   std::chrono::milliseconds stream_transaction_retry_interval;
+#ifdef MG_ENTERPRISE
+  bool is_coordinator{false};
+#endif
```

Comment:
It's probably easier to add the `is_coordinator` regardless of `MG_ENTERPRISE`.

Reply:
why? we have enterprise flags already